packages/console/src/consts/logs.ts

```diff
@@ -86,7 +86,9 @@ const webhookLogEventTitle: Record<string, Optional<string>> &
   'TriggerHook.PostSignIn': undefined,
 });
 
-export const logEventTitle: Record<string, Optional<string>> & Record<LogKey, Optional<string>> = {
+// Currently not determined where to put `jwtCustomizerLog`, not appending it for now.
+export const logEventTitle: Record<string, Optional<string>> &
+  Record<AuditLogKey | WebhookLogKey, Optional<string>> = {
```

Comment:
You can still use `LogKey`, and set the undetermined value to undefined.

Reply:
True, double checked and updated.